### PR TITLE
return nil if there are no tokens (i.e. no hosts available)

### PR DESCRIPTION
**Pull request by wyndhblb**
avoids panic should all cassandra nodes suddenly "disappear" (i.e. network partition or other)

### CODE REVIEW

File: token.go [hunk 1]
```diff
@@ -203,15 +203,22 @@ func (t *tokenRing) GetHostForToken(token token) *HostInfo {
 	if t == nil {
 		return nil
 	}
-
+	
+	l := len(t.tokens)
+	// no host tokens, no available hosts
+	if l == 0{
```
Comment (Zariel) [inline token.go]: Please gofmt this

**Comment by Zariel** [comment]
Can you gofmt the code and if not already please add yourself to the AUTHORS file

**Comment by Zariel** [comment]
Thanks for the contribution!

**Comment by Dieterbe** [comment]
thanks @wyndhblb we just had a panic due to this bug. looking fwd to roll out your fix :)

**Pull request merged**